hs_core/hydroshare/hs_bagit.py

```diff
@@ -21,20 +22,14 @@ def delete_bag(resource):
     # delete the resource bag
     istorage.delete('bags/{res_id}.zip'.format(res_id=res_id))
 
-def create_bag(resource):
-    """
-    Modified to implement the new bagit workflow. The previous workflow was to create a bag from the current filesystem
-    of the resource, then zip it up and add it to the resource. The new workflow is to delegate bagit and zip-up
-    operations to iRODS, specifically, by executing an iRODS bagit rule to create bagit file hierarchy, followed by
-    execution of an ibun command to zip up the bagit file hierarchy.
-
-    Note, this procedure may take awhile.  It is highly advised that it be deferred to a Celery task.
-
-    Parameters:
-    :param resource: (subclass of AbstractResource) A resource to create a bag for.
+    # delete the bags table
+    for bag in resource.bags.all():
+        try:
+            bag.delete()
+        except Exception as ex:
+            raise Exception(ex.message)
 
-    :return: the hs_core.models.Bags instance associated with the new bag.
-    """
+def create_bag_files(resource):
     from . import utils as hs_core_utils
     DATE_FORMAT = "YYYY-MM-DDThh:mm:ssTZD"
     istorage=IrodsStorage()
```

Comment:
Regarding:

    dest_prefix = getattr(settings, 'BAGIT_TEMP_LOCATION', '/tmp/hydroshare/')

What happens if the directory represented by dest_prefix does not exist?